test/puppetlabs/puppetdb/http/nodes_test.clj

```diff
@@ -69,57 +68,61 @@
 ;; TESTS
 
 (deftestseq node-queries
-  [[version endpoint] endpoints]
+  [[version endpoint] endpoints
+   method [:get :post]]
 
-  (let [{:keys [web1 web2 db puppet]} (store-example-nodes)]
+  (let [status-for-node' #(status-for-node method endpoint %)
```

Comment:
I'm guessing you're asking in reference to partial instead of an anonymous function? I don't tend to use partial much, but I do use -> and ->> pretty frequently